cvefeed/nvd/match_cpe.go

```diff
@@ -106,6 +106,10 @@ func (cm *cpeMatch) match(attr *wfn.Attributes, requireVersion bool) bool {
 	//	- matched attr without version
 	//  - didn't match version, or require version was set and version was *
 
+	if attr.Version == wfn.Any {
```

Comment:
as per the comment right above, shouldn't this be:
```
if !requireVersion && attr.Version == wfn.Any {
  return true
}
```

Reply:
Not really. RequireVersion should only apply to `cm` and not to `attr` (i.e. CPEs from vendor's feed, not user input).